imap_processing/spice/geometry.py

```diff
@@ -466,6 +466,46 @@ def get_rotation_matrix(
     return vec_pxform(from_frame.name, to_frame.name, et)
 
 
+def rotation_matrix_to_euler_angles(
+    rotation_matrix: npt.NDArray, axes: Union[list | None] = None
+) -> npt.NDArray:
+    """
+    Get the Euler angles from a rotation matrix.
+
+    This is a vectorized wrapper around `spiceypy.m2eul`
+    "Factor a rotation matrix as a product of three rotations about
+        specified coordinate axes."
+    https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/FORTRAN/spicelib/m2eul.html
+
+    Parameters
+    ----------
+    rotation_matrix : np.ndarray
+        Either single 2d rotation matrix of shape, `(3,3)` or 3d array
+        of rotation matrices of shape `(n, 3, 3)`.
+    axes : SpiceFrame
```

Comment:
Please update this to a list.
```suggestion
    axes : list, optional
```